### PR TITLE
RAB-1331: Add a sandbox helper

### PR DESCRIPTION
**Pull request by ghoussard**
Adding the Sandbox helper on every screen in the PIM for sanbox environments
image

### CODE REVIEW

File: src/Akeneo/Channel/back/Infrastructure/Symfony/Resources/config/datagrid/locale.yml
Comment (ValentinMumble) [inline src/Akeneo/Channel/back/Infrastructure/Symfony/Resources/config/datagrid/locale.yml:1]: This grid was unused

File: src/Oro/Bundle/PimDataGridBundle/Resources/config/data_sources.yml [hunk 1]
```diff
@@ -67,14 +67,6 @@ services:
         tags:
             - { name: oro_datagrid.datasource, type: pim_datasource_group }
 
-    pim_datagrid.datasource.locale:
```
Comment (ValentinMumble) [inline src/Oro/Bundle/PimDataGridBundle/Resources/config/data_sources.yml:70]: Unused

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/locale/index.yml
Comment (ValentinMumble) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/locale/index.yml:1]: Unused form extensions as the Locale index was rewritten in React

**Review by Beninho — APPROVED** [review]

**Review by masacc — APPROVED** [review]

File: src/Akeneo/Channel/back/Infrastructure/Symfony/Resources/config/datagrid/locale.yml
Comment (ValentinMumble) [inline src/Akeneo/Channel/back/Infrastructure/Symfony/Resources/config/datagrid/locale.yml:1]: Unused legacy datagrid

**Review by cemorin — APPROVED** [review]

**Review by guillaumefer-akeneo — APPROVED** [review]

**Pull request merged**